core/templates/mathjaxConfig.ts

```diff
@@ -1,4 +1,5 @@
 window.MathJax = {
+  // @ts-ignore
```

Comment:
Why is this ignored? Add comment explaining this.

Reply:
Added a comment in this thread itself. Can you take a look at that.

Reply:
Saw but still, this should be explained in the code.

Reply:
Done